openapi/mft/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["IT Operations/Message Brokers", "Cost/Freemium"]
+org = "ballerinax"
+name = "mft"
+icon = "icon.png"
+distribution = "2201.0.0"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.1.0"
```

Comment:
Since this is a new connector, version can be `1.0.0`

Reply:
Updated